addons/website/views/snippets/s_social_media.xml

```diff
@@ -2,25 +2,26 @@
 <odoo>
 
 <template id="s_social_media" name="Social Media">
-    <div class="s_social_media text-start o_not_editable" contenteditable="false">
+    <div class="s_social_media text-start o_not_editable" contenteditable="false" data-vxml="001">
+    <!-- TODO remove the contenteditable="true" ? -->
```

Comment:
@dmo-odoo it's the only `s_social_media` where I see `contenteditable="true"`, that's why I'm asking.

Reply:
If you remove `contenteditable="true"` here then the `h4` with text "Social Media" will not be editable anymore, is it ?

Reply:
IMO contenteditable="XXX" should be left to JS code,. Both, especially contenteditable="true" of course will be removed when leaving edit mode anyway. So when entering edit mode again, the contenteditable attribute is re-added where needed (either by the o_not_editable mechanism or your most recent mechanism `getContentEditableAreas`/`getReadOnlyAreas` (still think we'll need something better to unify this in the future, maybe OWL registries or something like that)).

So having contenteditable="false"/"true" in a snippet definition is neither good or bad but I would be in favor of not having those as already handled on the JS side. At least, in master, I would not change that in stable :wink:

Reply:
About the [comment above that I can't reply to](https://github.com/odoo/odoo/pull/118138#discussion_r1180376427)
I don't know when or where the `<h4>` is supposed to appear but I can't find it (nor edit it therefore).
Still, leaving those contenteditable attributes for now.

Reply:
Removed the comment. Still have not ever _seen_ an editable h4 (or h5 in some places) in the UI, just read about it in the code.